main/src/ca/uwaterloo/flix/language/ast/Type.scala

```diff
@@ -765,6 +796,15 @@ object Type {
     case _ => Type.Apply(Type.Cst(TypeConstructor.Complement, loc), tpe0, loc)
   }
 
+  /**
+    * Returns the type `Not(tpe0)`.
```

Comment:
Can this be used before kinding?

Reply:
Actually the "don't use before kinding" stuff ends up being redundant, since we now have `UnkindedType`